site/book/01-getting-started/02-quickstart.md

```diff
@@ -7,22 +7,40 @@ configuration using the underlying Git version control system.
 
 First, let's fetch the _kpt package_ from Git to your local filesystem:
 
+{{% hide %}}
+
+<!-- @makeWorkplace @verifyGuides-->
+```
+# Set up workspace for the test.
+TEST_HOME=$(mktemp -d)
+cd $TEST_HOME
+touch output.txt
+
+function expectedOutput() {
```

Comment:
@phanimarupaka we should avoid redefining these functions everywhere. Any common function and repetitive set up (e.g. create tmp file) should be extracted out.

Reply:
I created setupVerify.sh script which consists of all the functions. This script is executed first in the book and references README.md files to access the functions in the remaining files. I can modify the output of the tree command once it is fixed in #2030.